uber/model_checks.py

```diff
@@ -53,29 +57,48 @@ def group_paid(group):
 
 def _invalid_phone_number(s):
     if not s.startswith('+'):
-        return len(re.findall(r'\d', s)) != 10 or re.search(SAME_NUMBER_REPEATED, re.sub(r'[^0-9]','',s))
+        return len(re.findall(r'\d', s)) != 10 or re.search(c.SAME_NUMBER_REPEATED, re.sub(r'[^0-9]', '', s))
+
 
 def _invalid_zip_code(s):
     return len(re.findall(r'\d', s)) not in [5, 9]
 
-def attendee_misc(attendee):
-    if attendee.group_id and not attendee.first_name.strip() and not attendee.last_name.strip():
-        return
-
-    if COLLECT_EXACT_BIRTHDATE and attendee.birthdate == '':
-        attendee.birthdate = None # Prevent insertion errors for placeholder attendees
 
-    if not attendee.first_name or not attendee.last_name:
-        return 'First Name and Last Name are required'
-    elif attendee.placeholder:
-        return
-
-    if COLLECT_EXACT_BIRTHDATE and attendee.birthdate is None:
-        return 'Enter your date of birth.'
-    if COLLECT_EXACT_BIRTHDATE and attendee.birthdate > date.today():
-        return 'You cannot be born in the future.'
-
-    if COLLECT_FULL_ADDRESS:
+def ignore_unassigned_and_placeholders(func):
```

Comment:
This decorator is pretty good for MAGFest's purposes but I worry about it being a bit too inflexible. Would it not be better to, maybe, pass each validation through a decorator that checks it against a list of classes and validations that should be skipped for whatever parameters you define? Basically, I'd rather have one place to define what is considered "skippable" for what kind of attendee rather than changing which functions have a specific decorator.